libc/src/__support/math/CMakeLists.txt

```
@@ -775,6 +759,34 @@ add_header_library(
     libc.src.__support.common
 )
 
+add_header_library(
+  rsqrtf
+  HDRS
+    rsqrtf.h
+  DEPENDS
+    libc.hdr.errno_macros  
+    libc.hdr.fenv_macros  
+    libc.src.__support.FPUtil.cast  
+    libc.src.__support.FPUtil.fenv_impl  
+    libc.src.__support.FPUtil.fp_bits  
+    libc.src.__support.macros.optimization
```

- Nit: trailing whitespace (I wish GitHub would highlight it.).
- Missing `libc.src.__support.FPUtil.sqrt`.

```suggestion
    libc.hdr.errno_macros
    libc.hdr.fenv_macros
    libc.src.__support.FPUtil.cast
    libc.src.__support.FPUtil.fenv_impl
    libc.src.__support.FPUtil.fp_bits
    libc.src.__support.FPUtil.sqrt
    libc.src.__support.macros.optimization
```